.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
 - pip install -r requirements/tox.txt
 - pip install tox-travis
 script:
-- tox
+- tox -- -n 2 --durations=10
```

Comment:
It's a pytest option for listing the N slowest tests from the run: http://doc.pytest.org/en/latest/usage.html#profiling-test-execution-duration